app/views/admin/shared/forms/_name.html.haml

```diff
@@ -1,7 +1,5 @@
-.inst-box
-  %header
+.py-4.sm:grid.sm:grid-cols-3.sm:gap-4.sm:py-5.border-b.border-gray-300
```

Comment:
Is there anything we can do to make these long utility lists that we use often into some kind of helper or mix them together in a scss file?

Reply:
yep, I experimented with doing this but there was some reason why my first attempt wasn't working properly. Totally agree that we should re-factor these classes down to a single class, will keep trying.